guidelines.rst

```diff
@@ -0,0 +1,183 @@
+PEP: 9999
+Title: Guidelines for Python's C API
+Author: Petr Viktorin <encukou@gmail.com>,
+Discussions-To: https://discuss.python.org/c/c-api/30
+Status: Draft
+Type: Process
+Requires: 731
+Created: 20-Nov-2023
+Post-History: XXX
+
+.. pep-banner::
+
+   This is a **incomplete draft**, published for initial discussion.
+   It is likely to be amended substantially.
+
+.. Authors to be added:
+
+        Guido van Rossum <guido@python.org>,
+        Victor Stinner <vstinner@python.org>,
+        Steve Dower <steve.dower@python.org>,
+        Irit Katriel <irit@python.org>
+
+
+Abstract
+========
+
+TODO: Write the abstract last :)
+
+
+About this document
+===================
```

Comment:
It seems to me the _About this document_ section could function as an _Abstract_ :)

Reply:
Still, I think it should be moved there last :)